lib/readline.js

```diff
@@ -1331,7 +1331,8 @@ function codePointAt(str, index) {
   }
   return code;
 }
-exports.codePointAt = codePointAt;
+exports.codePointAt = util.deprecate(codePointAt,
+    'codePointAt() is deprecated. Use String.prototype.codePointAt');
```

Comment:
If it's undocumented, isn't is subject to disappearing at any time? (I realize someone could be using it)